src/yetibot/core/webapp/resolvers.clj

```diff
@@ -105,3 +107,10 @@
 (defn crons-resolver
   [context {:keys [] :as args} value]
   (db.cron/query {:query/identifiers identity}))
+
+(defn karmas-resolver
+  [context {:keys [] :as args} value]
+  (map (fn [{:keys [:user-id :score]}]
```

Comment:
Minor: it's idiomatic to eschew the `:` on the named keys inside the vector e.g. `{:keys [user-id score]}`

Reply:
Oh yeah!  Thanks for catching that :raised_hands: .